doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RainbowView.java

```diff
@@ -371,31 +397,89 @@ public void setCurrentValueLabelTextColor(@ColorInt final int textColor) {
         invalidate();
     }
 
-    public void setCurrentValueLabelTextSizePx(final float textSizePx) {
+    public void setCurrentValueLabelTextSizeSp(final float textSizeSp) {
+        final float textSizePx = spToPx(textSizeSp);
         final Paint newPaint = new Paint(getCurrentLevelTextPaint());
         newPaint.setTextSize(textSizePx);
         customCurrentValueLabelPaint = newPaint;
         invalidate();
     }
 
-    public void setExtremeValueLabelTypeface(@NonNull final Typeface typeface){
-        final Paint newPaint = new Paint(getExtremeLabelTextPaint());
-        newPaint.setTypeface(typeface);
-        customExtremeValueLabelPaint = newPaint;
+    public void setRangeLabelTextColor(@ColorInt final int color) {
+        final Paint newMinPaint = new Paint(getMinimumLabelTextPaint());
+        newMinPaint.setColor(color);
+        customMinimumLabelPaint = newMinPaint;
+
+        final Paint newMaxPaint = new Paint(getMaximumLabelTextPaint());
+        newMaxPaint.setColor(color);
+        customMaximumLabelPaint = newMaxPaint;
         invalidate();
     }
 
-    public void setExtremeValueLabelTextColor(@ColorInt final int color) {
-        final Paint newPaint = new Paint(getExtremeLabelTextPaint());
-        newPaint.setColor(color);
-        customExtremeValueLabelPaint = newPaint;
+    public void setRangeLabelTypeface(@NonNull final Typeface typeface){
+        final Paint newMinPaint = new Paint(getMinimumLabelTextPaint());
+        newMinPaint.setTypeface(typeface);
+        customMinimumLabelPaint = newMinPaint;
+
+        final Paint newMaxPaint = new Paint(getMaximumLabelTextPaint());
+        newMaxPaint.setTypeface(typeface);
+        customMaximumLabelPaint = newMaxPaint;
         invalidate();
     }
 
-    public void setExtremeValueLabelTextSizePx(final float textSizePx) {
-        final Paint newPaint = new Paint(getExtremeLabelTextPaint());
-        newPaint.setTextSize(textSizePx);
-        customExtremeValueLabelPaint = newPaint;
+    public void alignRangeLabelTextInward() {
```

Comment:
What are your thoughts on an `alignRangeLabelText()` or `setRangeLabelTextAlignment()` method that takes an enum as a parameter? We could use `INWARD`, `OUTWARD` and `CENTER` as options.